packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -214,17 +214,20 @@ export class PureDataObjectFactory<
 	 * @param context - The context being used to create the runtime
 	 * (the created object will have its own new context created as well)
 	 * @param initialState - The initial state to provide to the created data store.
+	 * @param loadingGroupId - experimental feature, that tries to delay load a data store. The service needs to support this feature, does not work for most services
 	 * @returns an object created by this factory. Data store and objects created are not attached to container.
 	 * They get attached only when a handle to one of them is attached to already attached objects.
 	 */
 	public async createChildInstance(
 		parentContext: IFluidDataStoreContext,
 		initialState?: I["InitialState"],
+		loadingGroupId?: string,
```

Comment:
One of earlier pieces of feedback we received - is it required to expose loadingGroupID, or can we use a boolean (load or not to load, even if under the cover we generate loadingGroup=uuid()).
I think ability to pass ID is required, as I think usage pattern will be - passing same ID for many data stores, but worth confirming.

Reply:
Offline, we confirmed that customers want to control loading group id. We cannot surface just use a boolean for them.